decidim-elections/config/locales/en.yml

```
@@ -105,6 +106,9 @@ en:
             title: New election
           publish:
             success: The election has been successfully published.
+          setup:
+            invalid: There was a problem setting up this election
+            success: Election setup successfully
```

Seems that the end of the phrase is missing 😁

```suggestion
            success: Election setup successfully sent to the Bulletin board
```